### PR TITLE
Controllers

### PR DESCRIPTION
**Pull request by KareemWilson**
## **In this branch, I did the following:** 

**1. Create routes for each following examples:** 

```
- /users
- /users/745
- /users/745/posts
- /users/745/posts/3
```

**2. Create `UsersController` and `PostsController`**
**3. Create Views for each `UsersController` and `PostsController` with simple placeholder**

### CODE REVIEW

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @KareemWilson  👍🏽,
## Good job so far!
### To Highlight 👇🏽
- No Linter error ✔️ 
- Good PR description 👍🏽 
- Good commit message(s) 😉

You still need to work on some issues to prepare your project for the final evaluation, but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR WORDS>


## Suggested changes
### Kindly fix your READEME MIT Lisence link, it currently broken 👍🏽
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you consider them as they can improve your code._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag (@rotimiazeez) me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class PostsController < ApplicationController
+  def show; end
+end
```
Comment (danmainah) [inline app/controllers/posts_controller.rb:3]: - [ ] [optional] @KareemWilson kindly add index method, its good practice to have a method that corresponds with the view eg a index page with index method

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class UsersController < ApplicationController
+  def initialize
+    super
+    @users = User.find(1, 2)
+  end
+
+  def show; end
```
Comment (danmainah) [inline app/controllers/users_controller.rb:7]: - [ ] [optional] @KareemWilson kindly add index method, its good practice to have a method that corresponds with the view eg an index page with index method

**Review by danmainah — APPROVED** [review]
Hi @KareemWilson , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Gitflow was correctly used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**